BUILDING.md

```diff
@@ -557,6 +557,23 @@ export CXX="ccache c++"        # add to ~/.zshrc or other shell config file
 export LDFLAGS="-fuse-ld=mold" # add to ~/.zshrc or other shell config file
 ```
 
+On Windows:
+
+Tips: follow <https://github.com/ccache/ccache/wiki/MS-Visual-Studio>, and you
+should notice that obj file will be bigger the normal one.
+
+First, install ccache, assume ccache install to c:\ccache, copy
+c:\ccache\ccache.exe to c:\ccache\cl.exe
+
+```powershell
+set TrackFileAccess=false
+set UseMultiToolTask=True
```

Comment:
I think you can add this two properties to the `props_4_ccache.props`, like in the docs: https://github.com/ccache/ccache/wiki/MS-Visual-Studio#usage-with-visual-studio

Reply:
You make a good point, I've been puzzled about using cmdline or config file quite some time.

Reply:
in fact, I am a little confuse about what talk here: <https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-cpp-builds?view=vs-2022>